### ISSUE TITLE
Upgrade to CI 4.5.5.

### ISSUE THREAD

**Issue by fazalimran** [Simpletine/CodeIgniter4-HMVC-Shield#24]
It is required to be updated to CI 4.5.5 as per CI documentation and recommendations.

**Comment by soumik141** [comment]
You can update your CI to the latest version by using the command "composer update". It will not affect the existing code.

**Comment by fazalimran** [comment]
Thanks you. Any chance of including all the adminlte plugins?

**Comment by soumik141** [comment]
@fazalimran yes, you can add the plugins manually to the "public\assets\simpletine\plugins" directory and use as per your requirement.
